src/BaGetter.Core/Metadata/IPackageMetadataService.cs

```diff
@@ -3,33 +3,30 @@
 using BaGetter.Protocol.Models;
 using NuGet.Versioning;
 
-namespace BaGetter.Core;
-
-/// <summary>
-/// The Package Metadata client, used to fetch packages' metadata.
-/// 
-/// See https://docs.microsoft.com/en-us/nuget/api/registration-base-url-resource
-/// </summary>
-public interface IPackageMetadataService
+namespace BaGetter.Core
```

Comment:
This makes the namespace block-scoped again, maybe merge (or rebase on) main, I guess the merge doesn't work for at least this file.